FEXCore/include/FEXCore/IR/IR.h

```diff
@@ -513,6 +513,12 @@ enum class SyscallFlags : uint8_t {
 
 FEX_DEF_NUM_OPS(SyscallFlags)
 
+enum NamedVectorConstant : uint8_t {
```

Comment:
Perhaps add a comment that this is linked to the array in CPUBackend.cpp as well and what it's for

Reply:
Added a comment.